pom.xml

```diff
@@ -6,7 +6,7 @@
 
   <groupId>edu.hm.hafner</groupId>
   <artifactId>analysis-model</artifactId>
-  <version>4.1.0-SNAPSHOT</version>
+  <version>5.0.0-SNAPSHOT</version>
```

Comment:
This should not be changed by you. ;)

Reply:
This is required for the pull request. If not updated the ci built will fail because of a changed code api.